hi-ml-cpath/src/health_cpath/utils/tiff_conversion_config.py

```diff
@@ -0,0 +1,117 @@
+#  -------------------------------------------------------------------------------------------
+#  Copyright (c) Microsoft Corporation. All rights reserved.
+#  Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+#  -------------------------------------------------------------------------------------------
+import logging
+import param
+from copy import deepcopy
+from health_azure.logging import logging_section
+from health_cpath.datasets.base_dataset import SlidesDataset
+from health_cpath.preprocessing.tiff_conversion import AMPERSAND, UNDERSCORE, ConvertWSIToTiffd, WSIFormat
+from health_cpath.utils.naming import SlideKey
+from monai.data.dataset import Dataset
+from pathlib import Path
+from torch.utils.data import DataLoader
+from tqdm import tqdm
+from typing import List, Optional
+from tifffile.tifffile import COMPRESSION
+
+
+class TiffConversionConfig(param.Parameterized):
+    image_key: SlideKey = param.ClassSelector(
+        class_=SlideKey,
+        default=SlideKey.IMAGE,
+        doc="The key of the image in the dataset. This is used to get the path of the src file."
+    )
+    src_format: WSIFormat = param.ClassSelector(
+        class_=WSIFormat, default=WSIFormat.NDPI, doc="The format of the source files. Default is NDPI."
```

Comment:
See my comment in tiff_conversion - this argument name is somewhat misleading, it is only the expected file extension?

Reply:
removed this param!